### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by charleshkang**
The README was looking a little bleak, so I added a bit of info on the project.

Maybe add a section that includes a thing on how to contribute?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+ - iOS 10.0
+
+ ## Installation
+ - Install [Cocoapods](http://guides.cocoapods.org/using/getting-started.html#installation).
+ - cd to directory and use `pod init` to create a Podfile
+ - Paste the below snippet into your Podfile
+
+```
+ Uncomment this line to define a global platform for your project
+platform :ios, '10.0'
```
Comment (vhart) [inline README.md]: why is this part of the Readme ?

Comment (charleshkang) [reply README.md]: removed!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+- Xcode 8
+- iOS 10.0
+
+## Installation
+- Fork/Clone the repo: [https://github.com/vhart/InterviewFlashCards.git](https://github.com/vhart/InterviewFlashCards.git)
+- Run `pod install` and open InterviewFlashCards.xcworkspace
+
+## Features
+- Choose between 3 different study topics
+- Question tracking to make sure you're understanding the topics and concepts
```
Comment (vhart) [inline README.md]: we don't actually track anything yet

File: README.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+- Solutions to questions within the app
+- Swipe through all questions and categories smoothly
```
Comment (vhart) [inline README.md]: we removed swipe temporarily remember?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
-# InterviewFlashCards
+# Interview Flash Cards
+An open-source project in the form of iOS technical, data structures, and algorithm questions in flashcard form.
```
Comment (vhart) [inline README.md]: `An open-source project designed to be a tool for iOS developers preparing for an interview. The project consists of data structure and algorithm questions, as well as iOS technical questions, all in flashcard form.`

Comment (vhart) [reply README.md]: maybe that's more understandable

File: README.md (hunk 2 above)
Comment (charleshkang) [reply README.md]: gotcha

File: README.md (hunk 3 above)
Comment (charleshkang) [reply README.md]: removed

File: README.md (hunk 4 above)
Comment (charleshkang) [reply README.md]: i like it! i was having trouble coming up with a project description.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,23 @@
-# InterviewFlashCards
+# Interview Flash Cards
+An open-source project designed to be a tool for iOS developers preparing for an interview. The project consists of data structure and algorithm questions, as well as iOS technical questions, all in flashcard form.
 
- Collection of iOS technical and data structures and algorithms questions in flashcard form. 
+## Pods
```
Comment (vhart) [inline README.md]: we can do without this section as the Podfile list will most likely change over time

Comment (charleshkang) [reply README.md]: gotcha

**Comment by mesbekmek** [comment]
LGTM

**Comment by charleshkang** [comment]
@vhart @mesbekmek if either of you could approve and merge that'd be great

**Review by vhart — APPROVED** [review]

**Pull request merged**